lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -873,14 +896,19 @@ void OpenCLFunction::execute() {
       setKernelArg(kernel, numArgs + 6,
                    ShapeNHWC(CC->getFilter()->getType()->dims()));
       if (isQuantized) {
-        setKernelArg(kernel, numArgs + 7, CC->getDest()->getType()->getOffset());
+        setKernelArg(kernel, numArgs + 7,
```

Comment:
Can you do the same trick with introduce a var for type and then using it to make the setKernelArg lines shorter?

Reply:
done